src/pivot-table/components/cells/DimensionTitleCell.tsx

```diff
@@ -110,9 +124,14 @@ const DimensionTitleCell = ({
         gridTemplateColumns: "1fr 24px",
         gridGap: "4px",
         alignItems: "center",
+        cursor: "pointer",
```

Comment:
This should be a conditional cursor and only be a pointer if `interactions.active` is `true`

Reply:
is this what we want right? 
(I'll fix menu icon in shared lib asap)

https://github.com/qlik-oss/sn-pivot-table/assets/29652890/84541520-dd33-4f09-9ba5-b0ed3e7a5465

Reply:
Yes 👍

Reply:
Promised PR is here to review: https://github.com/qlik-oss/nebula-table-utils/pull/46